### PR TITLE
AG8-138-cy-api-contentcreation-nt

### PR DESCRIPTION
**Pull request by nursena-tekoglu**

### CODE REVIEW

File: cypress/e2e/features/backend/nt_content-creation.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Feature: Content Creation
+
+Background: The user is logged in 
+    Given a valid refresh token is provided
+
+Scenario: The user verifies the grammar correcting function of the app
+    When the user verifies that he gets a successful response
+    Then the user verifies that the response is grammarly corrected version of the text
```
Comment (KubraSA) [inline cypress/e2e/features/backend/nt_content-creation.feature:8]: Try to read content from here or from a fixture file.

**Comment by nursena-tekoglu** [comment]
it is completed but will improve the code quality, reusability and maintainability.

File: cypress/integration/step_definitions/backend/nt_content-creation.cy.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+
+Then ('the user verifies that the response is {string} which is grammarly corrected version of the text' , () => {
+
+    cy.fixture('content_nt.json').then((contentData) => {
+        const originalText = contentData.content2;
+        const correctedText = expectedResponse;
+    
+        expect(correctedText).to.equal("I cook every day.");
```
Comment (KubraSA) [inline cypress/integration/step_definitions/backend/nt_content-creation.cy.js:53]: Try not to hard code. Read expected data from feature file or fixtures. You can add both in one fixture file. And manipulate from there

**Comment by nursena-tekoglu** [comment]
code completed.

**Pull request merged**